client/src/components/Search.tsx

```diff
@@ -1,146 +1,99 @@
+import Paper from '@mui/material/Paper';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
 import TextField from '@mui/material/TextField';
-import Tooltip from '@mui/material/Tooltip';
-import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
-import { useHistory } from 'react-router-dom';
+import SearchIcon from '@mui/icons-material/Search';
+import IconButton from '@mui/material/IconButton';
 
 import type { Theme } from '@mui/material/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
   searchBar: {
-    fontFamily: 'DM Sans',
+    gridArea: 'searchBar',
+    boxShadow: 'none',
+    margin: '50px auto',
+    width: '70%',
+  },
+  searchInput: {
     display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
     width: '100%',
-    borderRadius: '10px',
-    boxShadow: '0px 2px 4px 2px rgba(0, 0, 0, 0.25)',
-  },
-  select: {
-    background: theme.palette.background.default,
-    borderRadius: '10px',
-    flexBasis: '30%',
-    position: 'relative',
-    zIndex: 10,
-    borderBottom: 'none',
-    paddingInline: '20px',
-    [`&:before`]: {
-      border: 'none !important',
-    },
-    [`& svg`]: {
-      right: '26px',
-    },
-  },
-  textField: {
-    background: theme.palette.background.default,
-    borderRadius: '10px',
-    flexBasis: '70%',
-    color: 'red',
-    // height: '48px',
-    boxSizing: 'border-box',
-    border: '1px solid red',
-    [`& label`]: {
-      marginTop: '-8px',
-      marginLeft: '20px',
-    },
-    [`& div`]: {
-      marginTop: '0px !important',
-      [`&:before`]: {
-        borderBottom: 'none !important',
-      },
-      [`& input`]: {
-        height: '48px',
-        paddingLeft: '20px',
-      },
-    },
+    height: '70px',
   },
 }));
 
-function getTextFieldLabel(selectValue: string, searchText: string): string {
-  if (searchText) {
-    return '';
-  }
-
-  switch (selectValue) {
-    case 'All':
-      return 'Search all categories';
-    case 'Nonprofits':
-      return 'Search Nonprofits';
-    case 'Needs':
-      return 'Search Needs';
-    case 'Offers':
-      return 'Search Offers';
-    case 'Volunteer':
-      return 'Search Volunteer Openings';
-    default:
-      return 'Search all categories';
-  }
-}
-
 function Search() {
   const classes = useStyles();
-  const [selectedSearchCategory, setSelectedSearchCategory] = React.useState<string>('Needs');
-  const [searchText, setSearchText] = React.useState<string>('');
-
   const history = useHistory();
 
-  const handleDropdownChange = (event: SelectChangeEvent<string>) => {
-    setSelectedSearchCategory(event.target.value as string);
-  };
+  const searchParams = new URLSearchParams(history.location.search);
+  const querySearchText = searchParams.get('search');
+  const querySearchCategory = searchParams.get('category');
+
+  const [searchCategory, setSearchCategory] = React.useState<string>(
+    (querySearchCategory as string) || 'Needs',
```

Comment:
It will be safer to `null` coalesce - string casting is not checked during run-time
```suggestion
  const [searchCategory, setSearchCategory] = React.useState<string>(
   querySearchCategory ?? 'Needs',
```